modules/homa-lite/README.md

```diff
@@ -4,28 +4,26 @@
 The Homa Lite module allows users to upload some Staking currency, and mint some Liquid currency.
 
 ### Signed origin dispatchable calls
-* Request Mint: Request to upload Staking currency.  
+* Request Mint: Request to upload Staking currency to the relaychain via Xcm transfer.  
```

Comment:
`Xcm` should be `XCM`, the former one is naming conversion only for rustlang. Replace all of them to `XCM`.